packages/react/src/test/use-controller_test.tsx

```diff
@@ -251,12 +249,10 @@ suite('useController', () => {
         rerender = false;
         testController.host.requestUpdate();
       }
-      return React.createElement('div', {className: 'foo'}, [
-        `x:${x}, a:${testController.a}`,
-      ]);
+      return <div className="foo">{`x:${x}, a:${testController.a}`}</div>;
     };
 
-    const render = (props: any) => {
+    const render = (props: React.ComponentProps<typeof TestComponent>) => {
       act(() => {
         root.render(React.createElement(TestComponent, props));
```

Comment:
Optional: Why not use JSX here?

Reply:
Oops, I simply missed it. Thanks for catching!